### PR TITLE
Set up Travis CI #112

### LINKED ISSUE #112: Set up Travis CI
Although it is not a learning outcome of this level, we can still set up for the convenience of developers and contributors.

### PR DESCRIPTION
**Pull request by yamgent**
This PR will set up Travis by using gradle.

Fixes #112.

### CODE REVIEW

**Comment by yamgent** [comment]
@damithc Will need you to activate Travis on this repository for it to work.

**Comment by damithc** [comment]
> Will need you to activate Travis on this repository for it to work.

Done. Closed and reopened to trigger the build.

**Comment by damithc** [comment]
@m133225 can take a look?

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,9 @@
+language: java
+matrix:
+  include:
+    - jdk: oraclejdk8
+    
+addons:
+  apt:
+    packages:
+      - oracle-java8-installer
```
Comment (damithc) [inline .travis.yml]: Add carriage return here?

**Comment by yamgent** [comment]
@damithc Fixed the missing newline.

**Comment by damithc** [comment]
@m133225 

> Are we planning on moving the run-check scripts from level4 to this repository too (possibly in another PR)?

At this level, we can run the checkstyle checks and (if possible) also ensure I/O tests pass. Yes, as separate PRs.

**Pull request merged**